### ISSUE TITLE
redirectToHTTPS is not a function

### ISSUE THREAD

**Issue by tbiinfotech** [SegFaultx64/express-http-to-https#4]
I am working on node js and angular2 application. 
`const redirectToHTTPS = require('express-http-to-https');
app.use(redirectToHTTPS([/localhost:(\d{4})/], [/\/insecure/]));`

It is giving error to me as redirectToHTTPS is not a function.

**Comment by bbonamin** [comment]
@tbiinfotech seems it's now a named export.

Therefore you should use:

```javascript
const redirectToHTTPS = require("express-http-to-https").redirectToHTTPS;
app.use(redirectToHTTPS([/localhost:(\d{4})/], [/\/insecure/]));
```

**Comment by SegFaultx64** [comment]
Oops, thanks for pointing out the mistake in the README. Let me change that. 

@bbonamin is 100% correct, it is now a named export.

**Comment by jonathanfishbein1** [comment]
I'm still having some trouble with an issue that seems similar

I have `var redirectToHTTPS = require("express-http-to-https").redirectToHTTPS;`

generating the following error

> Application has thrown an uncaught exception and is terminated:
SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (D:\home\site\wwwroot\server.js:6:23)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
Application has thrown an uncaught exception and is terminated:
SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (D:\home\site\wwwroot\server.js:6:23)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)

**Comment by SegFaultx64** [comment]
I will take a look at this ASAP

**Comment by SegFaultx64** [comment]
Hi @jonathanfishbein1 this should be fixed by the new version 1.1.3 I just published to NPM. Please let me know if you are still having issues.

Best,

Max
